python/res/enkf/row_scaling.py

```diff
@@ -103,3 +111,52 @@ def exp_decay(grid, pos, r0, data_index):
 
         for index in range(target_size):
             self[index] = func(index)
+
+    def assign_vector(self, scaling_vector):
```

Comment:
Why isn't `assign_vector` implemented here, in the python layer, as a specialized call to `assign`?